.cirrus.yml

```diff
@@ -90,23 +94,28 @@ task:
 
 windows_task:
   auto_cancellation: true
-  
+
   windows_container:
     image: cirrusci/windowsservercore:cmake
     os_version: 2019
-    
+
+  matrix:
+    - only_if: $CIRRUS_PR != ""
+    - only_if: $CIRRUS_PR == ""
```

Comment:
can you comment the desired behavior for these conditionals?

Reply:
done